### DISCUSSION TITLE
How to enable JIT mode on TailwindCSS?

### DISCUSSION THREAD

**Discussion by vivgui** [uicrooks/shopify-theme-lab#85] [Q&A]
How can I enable JIT mode for TailwindCSS? I try adding `mode: 'jit'` to the config but that doesn't work for some reason.

Btw, great starter theme! I use it for all my new shopify projects now.

**Comment by sergejcodes** [comment]
Hi 👋,

the mode setting you mentioned should be right. I just tried it out on the latest Theme Lab version and everything seems to work. 

**Troubleshooting steps**
1. stop any running npm tasks
2. make sure you have the latest TailwindCSS version installed
3. add jit mode to `tailwind.config.js` and make sure the purge content path points to the right files:
```js
...

module.exports = {
  mode: 'jit',
  ...
  purge: {
    enabled: process.env.NODE_ENV === 'production',
    content: [
      path.resolve(__dirname, '**/*.{js,vue}'),
      path.resolve(__dirname, '../shopify/**/*.liquid')
    ]
  }
}
```
4. run `npm run start`

Here are jit arbitrary styles in action:

![screenshot](https://user-images.githubusercontent.com/68379148/120064415-46b1f200-c06c-11eb-83f9-84385a1ffc9b.png)

**Comment by vivgui** [comment]
Ah, thank you! My problem could be that I'm on a slightly old version, I'll look into that.
